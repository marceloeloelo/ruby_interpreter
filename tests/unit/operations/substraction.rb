0
puts(0)

0.123
puts(0.123)

-1
puts(-1)

-1.23
puts(-1.23)

1-1
puts(1-1)

1-2.3
puts(1-2.3)

1.2-3
puts(1.2-3)

1.2-3.4
puts(1.2-3.4)
