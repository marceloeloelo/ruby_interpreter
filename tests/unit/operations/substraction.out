0
0.123
-1
-1.23
0
-1.3
-1.8
-2.2
